scio-core/src/main/scala/com/spotify/scio/coders/KryoAtomicCoder.scala

```diff
@@ -102,7 +102,6 @@ private[scio] class KryoAtomicCoder[T] extends AtomicCoder[T] {
     }
   }
 
-  private val logger = LoggerFactory.getLogger(this.getClass)
```

Comment:
Replace with `import KryoAtomicCoder.logger` so the `logger.*` lines can stay the same?